### PR TITLE
Make background image responsive for mobile for CodeRadio

### PR DESCRIPTION
**Pull request by Stev24**
Checklist:

- [X ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X ] My pull request has a descriptive title (not a vague title like `Update index.md`)

Closes https://github.com/freeCodeCamp/freeCodeCamp/issues/41072

I could not fixe for bigger screens than FHD because I have dont have one.

### CODE REVIEW

File: src/css/App.css [hunk 1]
```diff
@@ -1,14 +1,14 @@
 @import url('https://fonts.googleapis.com/css?family=Lato:400,700&display=swap');
 
-/* Globals */
+/* Globals Steven Ribeiro*/
```
Comment (ieahleen) [inline src/css/App.css:3]: Please avoid leaving a signature in the codem the PR will be attributed to you

**Review by ieahleen — CHANGES_REQUESTED** [review]

File: src/css/App.css (hunk 1 above)
Comment (Stev24) [reply src/css/App.css]: ok thanks

**Comment by ahmadabdolsaheb** [comment]
Thanks for submitting this pr.
The image is larger than it is supposed to for smaller screens.(13inch)
The aim is to display as much as the illustration as possible without showing empty areas.
For mobile phone dimensions, the illustrations is not supposed to be displayed.
The left image is the current state on coderadio.
Screen Shot 2021-10-06 at 10 35 23 AM

**Comment by ahmadabdolsaheb** [comment]
You could emulate larger screen using the browser's zoom functionality. (it is almost the same)
So we would like to show as much of the illustration as possible at any given ratio.

**Comment by ahmadabdolsaheb** [comment]
Closing due to inactivity. Please let me know if this pr has been closed by mistake.

**Pull request closed without merging**